mitodl/ocw-hugo-themes
# remove zip file existence check

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes https://github.com/mitodl/ocw-hugo-themes/issues/1076

#### What's this PR do?
This PR removes some code that checks to make sure that a ZIP file exists at the expected URL before generating a link button pointing to it on the download page. This functionality was introduced in https://github.com/mitodl/ocw-hugo-themes/pull/960 and https://github.com/mitodl/ocw-hugo-themes/pull/979 as a way to prevent the link to the download button from being generated at build time if the file did not exist yet. The code is fragile, and relied on the existence of a bug in Hugo that has since been fixed - https://github.com/gohugoio/hugo/issues/10604. We no longer need this check, as since https://github.com/mitodl/ocw-studio/pull/1630 is part of the single course publishing pipeline now, the ZIP is guaranteed to be generated and put in place before the first online build of a course is run.

#### How should this be manually tested?
 - Run `yarn start course`
 - Visit http://localhost:3000 and click the download course button in the course info drawer
 - On the download page, you should see a button to download the course that links to a ZIP file
 - The link won't work in the local dev server, but it should look something like `https://localhost:3000/10.302-fall-2004.zip`
